docs/userGuide/syntax/annotations.md

```diff
@@ -0,0 +1,110 @@
+{% from "userGuide/components/advanced.md" import slot_info_trigger %}
+
+## Annotations
+
+Annotation components allows you to easily add annotations over any image.
+
+### Introduction
+
+Annotation wrappers (`<annotate>`) are used in conjunction with annotation
+points (`<a-point>`).
+
+Annotation wrappers are used to hold the image and set the width and height of
+said image.
+
+Annotation points are used to define the position, text and style of each point
+within the image. In use, the annotation points are directly inserted between
+the annotate wrappers (`<annotate>...</annotate>`).
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">html</variable>
+<variable name="code">
+
+<annotate src="https://www.researchgate.net/profile/Chaiwat-Sakul/publication/228949121/figure/fig2/AS:300654661783574@1448693065750/The-previous-square-root-circuit-1.png" width="500">
+  <!-- Minimal Point -->
+  <a-point x="25%" y="25%" content="Lorem ipsum dolor sit amet" />
+  <!-- Customize Point Label (default is empty) -->
+  <a-point x="50%" y="25%" content="Lorem ipsum dolor sit amet" label="1a"/>
+  <!-- Customize Point Header (default is empty) -->
+  <a-point x="75%" y="25%" content="Lorem ipsum dolor sit amet"  header="Lorem ipsum"/>
+  <!-- Customize Point Size (default size is 40px) -->
+  <a-point x="25%" y="50%" content="Lorem ipsum dolor sit amet"  size="60"/>
+  <!-- Customize Point Color (default color is green) -->
+  <a-point x="50%" y="50%" content="Lorem ipsum dolor sit amet"  color="red"/>
+  <!-- Customize Point Opacity (default opacity is 0.3) -->
+  <a-point x="75%" y="50%" content="Lorem ipsum dolor sit amet"  opacity="0.7"/>
+</annotate>
+</variable>
+</include>
+
+**Using trigger for Annotate Point** <br>
+
+Similar to popovers, annotation points also support different types of triggers for users with different needs.
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">html</variable>
+<variable name="code">
+
+<annotate src="https://www.researchgate.net/profile/Chaiwat-Sakul/publication/228949121/figure/fig2/AS:300654661783574@1448693065750/The-previous-square-root-circuit-1.png" width="500">
+  <!-- Default Trigger (click)-->
+  <a-point x="33%" y="50%" content="Lorem ipsum dolor sit amet" />
+  <!-- Set Trigger to hover focus -->
+  <a-point x="66%" y="50%" content="Lorem ipsum dolor sit amet" trigger="hover focus"/>
+</annotate>
+</variable>
+</include>
+
+<br>
+
+**Displaying content as legends in Annotate Point** <br>
+
+Annotate point also allows users to display its content as a legend below the diagram. However, this only works when a label is provided for the point.
```

Comment:
Maybe can also mention that when a `legend` attribute is used, the annotation will not be clickable.

Reply:
> Is it possible to make the annotation point responsive?

Actually thanks to the help of @jovyntls and her godlike css skillz, it is now responsive, I'm still working on minor bug fixes and documentation cleanups!